runtime/parachains/src/lib.rs

```diff
@@ -21,6 +21,8 @@
 //! of the other modules.
 
 #![cfg_attr(not(feature = "std"), no_std)]
+// `construct_runtime!` does a lot of recursion and requires us to increase the limit to 256.
+#![recursion_limit = "256"]
```

Comment:
sidenote I thought construct_runtime didn't use that much recursion anymore actually.

Reply:
Removed this now and it seems to work okay now.